migrations/1756371866.sh

```diff
@@ -0,0 +1,3 @@
+echo "Reload notifications"
+cp ~/.local/share/omarchy/default/mako/core.ini ~/.config/mako/
```

Comment:
I don't think this is right? We are not supposed to copy the core. Only the current theme one.

Reply:
Actually, don't even need to copy current theme. That's already in the right place.